main.py

```diff
@@ -1,9 +1,185 @@
+        c = conn.cursor()
+        c.execute(create_table_sql)
+    except Error as e:
+        print(e)
+
+
+def create_task(conn, task):
+    """
+    Create a new task
+    :param conn:
+    :param task:
+    :return:
+    """
+
+    sql = ''' INSERT INTO tasks(name,status)
+              VALUES(?,?) '''
+    cur = conn.cursor()
+    cur.execute(sql, task)
+    conn.commit()
+    return cur.lastrowid
+
+
+def main_db_upload(upload):
+    # create a database connection
+    conn = create_connection(database)
+
+    sql_create_tasks_table = """CREATE TABLE IF NOT EXISTS tasks (
+                                        id integer PRIMARY KEY,
+                                        name text NOT NULL,
+                                        status bool NOT NULL
+                                    );"""
+    # create tables
+    if conn is not None:
+
+        # create tasks table
+        create_table(conn, sql_create_tasks_table)
+    else:
+        print("Error! cannot create the database connection.")
+    with conn:
+        # create tasks
+        new_task = (upload.name, upload.status)
+        create_task(conn, new_task)
+
+
+def main_db_download(status):
+    # create a database connection
+    conn = create_connection(database)
+
+    sql_create_tasks_table = """CREATE TABLE IF NOT EXISTS tasks (
+                                        id integer PRIMARY KEY,
+                                        name text NOT NULL,
+                                        status bool NOT NULL
+                                    );"""
+    # create tables
+    if conn is not None:
+
+        # create tasks table
+        create_table(conn, sql_create_tasks_table)
+    else:
+        print("Error! cannot create the database connection.")
+    with conn:
+        # select tasks
+        select_task_by_status(conn, status)
+
+
+def select_all_tasks(conn):
+    """
+    Query all rows in the tasks table
+    :param conn: the Connection object
+    :return:
+    """
+    cur = conn.cursor()
+    cur.execute("SELECT * FROM tasks")
+
+    rows = cur.fetchall()
+
+    for row in rows:
+        print(row)
+
+
+def select_task_by_status(conn, status):
+    """
+    Query tasks by status
+    :param conn: the Connection object
+    :param status:
+    :return:
+    """
+    cur = conn.cursor()
+    cur.execute("SELECT * FROM tasks WHERE status=?", (status,))
+
+    rows = cur.fetchall()
+
+    for row in rows:
+        print(row)
+
+
+def main_menu():
+    print("1: Wyświetlenie listy zrobionych zadań")
+    print("2: Wyświetlenie listy zadań do wykonania")
+    print("3: Dodaj zadanie")
+    print("0: Wyjście")
+    choice = input('Wybierz menu: ')
+    if choice == "1":
+        complete_task()
+    elif choice == "2":
+        incomplete_task()
+    elif choice == "3":
+        new_task()
+    elif choice == "0":
+        exit()
+    else:
+        print("Błędny wybór")
+
+
+def return_to_menu():
+    choice = input('Wyjście z programu [T/N]? ')
```

Comment:
W linuksie jest tak, że jak dajesz wybór, to wartość domyślna jest wielką literą, a pozostałe wybory są małymi. Czyli w np
`[T/n]` i jesli użytkownik nic nie wpisze i wciśnie enter, to potraktowane to zostanie jak byś wybrał opcją, która jest wielką literą, czyli tutaj `T`

Reply:
dobra praktyka, będę pamiętał. Poprawiono i dodano funkcjonalność wciśnięcie samego entera

Reply:
def return_to_menu():
    choice = input('Wyjście z programu [T/n]? ')
    if choice.lower() == "n":
        main_menu()
    elif choice.lower() == "t" or choice == "":

Reply:
``` python
def return_to_menu():
    choice = input('Wyjście z programu [T/n]? ')
    if choice.lower() == "n":
        main_menu()
    elif choice.lower() == "t" or choice == "":
```
Można też zamiast sprawdzać czy jest "", to 
``` python
len(choice) == 0
```

Reply:
![obraz](https://user-images.githubusercontent.com/21276887/111083626-ba813b80-850e-11eb-8d77-cda3c60ed926.png)

Reply:
len(choice) wygląda bardziej profesjonalnie. zmieniłem.

Reply:
``` python
def new_task():
    print("Podaj nazwę nowego zadania: ")
    new = task(input(), False)
    main_db_upload(new)
    choice = input("Chcesz dodać kolejne zadanie [T/n]? ")
    if choice.lower() == "n":
        main_menu()
    elif choice.lower() == "t" or len(choice) == 0:
        new_task()

Reply:
```
def new_task():
    print("Podaj nazwę nowego zadania: ")
    new = task(input(), False)
    main_db_upload(new)
    choice = input("Chcesz dodać kolejne zadanie [T/n]? ")
    if choice.lower() == "n":
        main_menu()
    elif choice.lower() == "t" or len(choice) == 0:
        new_task()

Reply:
teraz czaje - fajne opcje